src/apigateway/activation.ts

```diff
@@ -6,15 +6,26 @@
 import * as vscode from 'vscode'
 import { RestApiNode } from './explorer/apiNodes'
 import { invokeRemoteRestApi } from './commands/invokeRemoteRestApi'
+import { copyUrlCommand } from './commands/copyUrl'
+import { ExtContext } from '../shared/extensions'
 
 /**
  * Activate API Gateway functionality for the extension.
  */
 export async function activateApiGateway(activateArguments: {
```

Comment:
nit: The common convention for all activation.ts files is to name the activation method `activate`

Reply:
Then they have to be disamgiuated in import? do we really want to do that?

Reply:
Does the disambiguation cause much concern? The nice thing about using the same name is it takes the guesswork out of each domain's entrypoint. Down the road it might be reasonable to generalize all of the activations somehow, which we'd want consistent naming for too.

Reply:
> Then they have to be disamgiuated in import? do we really want to do that?

That can be avoided with `import * as foo ...` followed by `foo.activate()`. IMO we should use that approach more often in general.

Alternatively if we are going to require all functions to be globally disambiguated without their module/class qualifier, then we should not have so many classes/modules...